src/marqo/_httprequests.py

```diff
@@ -129,6 +182,19 @@ def __validate(
         except requests.exceptions.HTTPError as err:
             convert_to_marqo_web_error_and_raise(response=request, err=err)
 
+    def calculate_backoff_sleep(self, attempt: int, cap: int) -> float:
+        """Calculates the backoff sleep time for a given attempt
+
+        Args:
+            attempt (int): the attempt number
+
+        Returns:
+            float: the backoff sleep time
+        """
+        return min(
+            cap * 1000, # convert to milliseconds
+            (2 ** attempt) * 10 # start at 10ms for first attempt
```

Comment:
Is there any particular basis for doing time intervals of 10ms, 20ms, 30ms, etc?

Reply:
Yep, this reasoning makes sense to me, thanks!